WebAPI/Program.cs

```diff
@@ -16,6 +16,10 @@
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
 
+builder.Services.AddLogging(loggingBuilder => {
+    loggingBuilder.AddFile("app.log", append: true);
+});
```

Comment:
If you've added new logger, should've replaced the places where we/you have used old one... Probably will cause merge conflict so do after merging your other PR

Reply:
I would also probably suggest using a more popular nugget: [NLoh](https://nlog-project.org/). But not sure, if yours somewhy better, let me know :)

Reply:
NLog requires a lot of configuration like additional xml file (nlog.config)
Only difference, that I don't like in NLog is that it overrides old logs.
Current implementation appends logs to one file saving old ones.

Reply:
Does the current logger also create files by date? Since if we will keep appending to the same file for a month, then finding older logs will be impossible as well as opening file will be a struggle for PC :)

Reply:
Because from the code itself I can't see any settings for file names, so I assume it will always use one single file.

Reply:
> If you've added new logger, should've replaced the places where we/you have used old one... Probably will cause merge conflict so do after merging your other PR

Ohh it uses same ILogger, but it somehow bridges to a file, so it works with same implementation

Reply:
> Does the current logger also create files by date? Since if we will keep appending to the same file for a month, then finding older logs will be impossible as well as opening file will be a struggle for PC :)

Updated. Now it saves log files by dates in logs folder. It also appends logs if it is same date.